README.md

```diff
@@ -18,4 +18,29 @@ Generate outputs the following data:
 
 Generate consists of several components:
 - download list creator: Creates list of files to download (search and download from OBPG).
-- downloader: Downloads files from lists created by the download list creator.
\ No newline at end of file
+- downloader: Downloads files from lists created by the download list creator.
+- combiner: Combines downloaded files into a single NetCDF file.
+- processor: Processes combined files into final L2P granule NetCDF file.
+- error_handler: Handles AWS Batch job failures by logging and notification.
+
+## aws infrastructure
+
+The Generate workflow includes the following AWS services:
+- AWS Batch compute environment with launch template and user-data script, job queue, and scheduling policy for each dataset.
+- Elastic file system for the following components: downloader, combiner, processor.
+- IAM roles and policies for Batch and ECS permissions.
+- S3 bucket to hold final L2P output.
+- Security groups to support EFS network traffic in VPC.
+
+## terraform 
+
+Deploys AWS infrastructure and stores state in an S3 backend using a DynamoDB table for locking. The top-level `terraform` directory contains AWS infrastructure that applies to all components. Each component may have additional terraform files for deploying AWS resources, see each components `README.md` for details.
```

Comment:
I don't see any definition of the infrastructure for "using a DynamoDB table for locking"? I see a table name being passed in as a variable but nothing else. How does this work?

Reply:
The DynamoDB table and S3 backend are defined separately which I don't think makes sense at this time. I will incorporate their definition into the top-level Generate Terraform files.

Reply:
I'm not familiar with using DynamoDB for locking the backend. We've generally just used the S3 bucket to track terraform state (but not locking with Dynamo). We do have a common infrastructure repository that may be a better place for this. Do the Cumulus accounts use this locking feature?

Reply:
I don't know if the Cumulus accounts use the locking feature, maybe @skorper can let us know? Happy to remove this so that Generate infrastructure is consistent with other programs.